recipes/cuda-api-wrappers/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+#include <cuda/undefine_specifiers.hpp>
+
+
+int main() {
+
```

Comment:
Try consuming something from header. You could generate a version, for instance: https://github.com/eyalroz/cuda-api-wrappers/blob/master/src/cuda/api/versions.hpp#L115

Reply:
That's not possible, there is a dependency to cuda. This recipe shouldn't be merged until we have a cuda recipe.